src/gui/systray.cpp

```diff
@@ -568,7 +568,11 @@ void Systray::showTalkMessage(const QString &title, const QString &message, cons
 
 void Systray::setToolTip(const QString &tip)
 {
+#ifdef Q_OS_WIN
     QSystemTrayIcon::setToolTip(tr("%1: %2").arg(Theme::instance()->appNameGUI(), tip));
+#else
+    QSystemTrayIcon::setToolTip(tip);
+#endif
```

Comment:
I tried to follow the design discussion
still I would really prefer not to have conditional compilation there
is it really needed to differentiate between platforms ?

Reply:
I thought that on Windows it would make more sense to still have the application name in there as well, but I'm also fine with removing this overridden method altogether.

@jancborchardt, any thoughts on this?